modules/system-info/reporters/wordpress.php

```diff
@@ -74,7 +74,7 @@ public function get_fields() {
 	 */
 	public function get_memory_limit() {
 		$result = [
-			'value' => ini_get( 'memory_limit' ),
+			'value' => (string) WP_MAX_MEMORY_LIMIT, // Use WordPress max memory limit, since its dynamic.
```

Comment:
Are you sure the right value is  `WP_MAX_MEMORY_LIMIT` and not `WP_MEMORY_LIMIT`?

Reply:
We need to add all of them, in the past we already used the `WP_MEMORY_LIMIT` but on some servers it was different from the `ini_get( 'memory_limit' )`. https://github.com/elementor/elementor/commit/02f81ced71b7ce3c65e5d74377206afb014b6352
@iNewLegend please add the 3 values: `get_ini_memory_limit, get_wp_memory_limit, and, get_wp_max_memory_limit`
See https://github.com/WordPress/WordPress/blob/master/wp-includes/default-constants.php#L37

Reply:
`WP_MAX_MEMORY_LIMIT` is the most accurate since it provide the max memory limit for admin and see here:
https://github.com/WordPress/WordPress/blob/master/wp-includes/default-constants.php#L51
its favor the `wp_is_ini_value_changeable( 'memory_limit' )`.
`WP_MEMORY_LIMIT` is for non admin users.

In order to understand it, please provide me an example where `WP_MAX_MEMORY_LIMIT` is not correct.

Reply:
Maybe just do like this:
```PHP
$memory_limit = min( [ WP_MAX_MEMORY_LIMIT, WP_MEMORY_LIMIT, ini_get( 'memory_limit' ) ] );
```

Reply:
Why?